lib/ActivityStream.jsm

```diff
@@ -240,7 +240,7 @@ const PREFS_CONFIG = new Map([
   }],
   ["discoverystream.endpoints", {
     title: "Endpoint prefixes (comma-separated) that are allowed to be requested",
-    value: "https://getpocket.cdn.mozilla.net/",
+    value: "https://getpocket.cdn.mozilla.net/,https://spocs.getpocket.com",
```

Comment:
This probably should have a trailing slash, otherwise someone could register something like `https://spocs.getpocket.comdummydomain.com`

Reply:
:+1:

Reply:
Updated, thanks for catching this, was a big one.